tests/services/test_file_workflow_service.py

```
@@ -1177,4 +1179,160 @@ def test_record_file_in_db_legacy_path(monitored_path, db_session, tmp_path):
     assert record.cold_storage_path == str(dest_file)
 
 
+class MockRemoteBackend:
+    def backend_name(self) -> str:
+        return "gdrive"
+
+    def build_reference(self, location, path) -> str:
```

Ruff (0.15.15)

[warning] Unused method argument: `location`

(ARG002)